app/src/main/res/layout/submission_item.xml

```diff
@@ -10,7 +10,8 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:padding="10dp"
-        android:textSize="24sp"
+        android:textAppearance="?attr/textAppearanceHeadline5"
+        android:textColor="?android:textColorSecondary"
```

Comment:
I recommend you to use material design attributes.
You can change them by overriding the style in `styles.xml`.
In that way you won't have harcoded values in the xmls and it will be easier to change in the future

Reply:
Added as an issue to address separately 👍